code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1146,6 +1146,15 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_desc = "A drink from Valhalla."
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
+/datum/reagent/consumable/ethan/mead/on_mob_life(mob/living/carbon/M, delta_time, times_fired)
+    holder.add_reagent(/datum/reagent/consumable/sugar, 3 * REM * delta_time)
```

Comment:
did you intent this

Reply:
Forgot about it, I will see what comments think about it and will go from there whether it stays or not.

Reply:
did you combat mode this*